.travis.yml

```diff
@@ -19,9 +18,13 @@ matrix:
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.*
+      env: DEPENDENCIES="symfony/lts:^2"
+    - php: 7.1
+      env: DEPENDENCIES="symfony/lts:^3"
+    - php: 7.1
+      env: STABILITY="dev"
```

Comment:
As Symfony 4 has been released, we can now test against a stable version.

Reply:
I updated that :). But let's try not to look too hard at the Travis file - it can be changes easily later. Plus, I think I *added* the STABILITY="dev" by someone's request - we could bikeshed this file for ages (fortunately, we're already doing that in that symfony-docs PR!)